src/main/java/seedu/address/ui/EventContactDisplay.java

```diff
@@ -34,6 +36,9 @@ public class EventContactDisplay extends UiPart<Region> {
     @FXML
     private StackPane eventListPanelPlaceholder;
 
+    @FXML
+    private StackPane reminderTaskListPanelPlaceholder;
```

Comment:
wow this is a really long variable name.....

Reply:
Changed it to `taskPanelPlaceholder`